pkg/agent/agentclient/client.go

```diff
@@ -264,8 +264,10 @@ func (a *AgentClient) proxyToRemote(conn net.Conn, connID int64) {
 		for {
 			n, err := conn.Write(d[pos:])
 			if err == nil {
+				klog.Infof("[connID: %d] write last %d data to remote", connID, n)
 				break
 			} else if n > 0 {
+				klog.Infof("[connID: %d] write %d data to remote", connID, n)
```

Comment:
err != nil in this case correct? Might be worth logging the err.

Reply:
Done.